tests/fs.md

```diff
@@ -1011,3 +1017,23 @@ Exception: Failure "Simulated error".
 +"/" / "" = "/"
 - : unit = ()
 ```
+
+```ocaml
```

Comment:
To run the test make sure you run `dune runtest` and `dune promote` the output back to this file.